octobot/logger.py

```diff
@@ -259,12 +269,12 @@ async def balance_profitability_callback(
 
 
 async def trades_callback(
-    exchange: str,
-    exchange_id: str,
-    cryptocurrency: str,
-    symbol: str,
-    trade: dict,
-    old_trade: bool,
+        exchange: str,
+        exchange_id: str,
+        cryptocurrency: str,
+        symbol: str,
+        trade: dict,
+        old_trade: bool,
```

Comment:
why these double indents ?

Reply:
It's probably a pycharm auto refactor, I'll revert it :+1: